blocks/library/text/index.js

```diff
@@ -44,6 +47,18 @@ registerBlockType( 'core/text', {
 					/>
 				</BlockControls>
 			),
+			focus && (
+				<InspectorControls key="inspector">
+					<div className="blocks-text__drop-cap" style={ { display: 'flex', justifyContent: 'space-between' } }>
```

Comment:
Just speed considering all these controls will be superseded by components in following iterations, properly absorbing the styles.